src/modules/navigation/SideNav.vue

```diff
@@ -46,9 +58,9 @@ export default {
     const showOrHide = computed(() =>
       store.state.isSideNavCollapsed ? "side-nav__hide" : "side-nav__show"
     );
-    const userIsAdmin = computed(() => 
-        store.state.user_data 
-        ? store.state.user_data.roles.includes('admin') 
+    const userIsAdmin = computed(() =>
+        store.state.user_data
```

Comment:
try  `store.state.user_data?.roles?.includes('admin')` instead of  `store.state.user_data ?store.state.user_data.roles.includes('admin') : false`